app/bundles/SmsBundle/Controller/Api/SmsApiController.php

```diff
@@ -50,4 +60,57 @@ public function receiveAction()
         // Return an empty response
         return new Response();
     }
+
+    /**
+     * @param $id
+     * @param $contactId
+     *
+     * @return JsonResponse|Response
+     */
+    public function sendAction($id, $contactId)
+    {
+        if (!$this->get('mautic.sms.transport_chain')->getEnabledTransports()) {
+            return new JsonResponse(json_encode(['error' => ['message' => 'SMS transport is disabled.', 'code' => Codes::HTTP_EXPECTATION_FAILED]]));
+        }
+
+        $message  = $this->model->getEntity((int) $id);
+
+        if (is_null($message)) {
+            return $this->notFound();
+        }
+
+        $contact  = $this->checkLeadAccess($contactId, 'edit');
+
+        if ($contact instanceof Response) {
+            return $this->accessDenied();
+        }
+
+        $this->get('monolog.logger.mautic')->addDebug("Sending SMS #{$id} to contact #{$contactId}", ['originator'=>'api']);
+
+        try {
+            $response = $this->model->sendSms($message, $contact, ['channel' => 'api'])[$contact->getId()];
+        } catch (\Exception $e) {
+            $this->get('monolog.logger.mautic')->addError($e->getMessage(), ['error'=>(array) $e]);
+
+            return new Response('Interval server error', Codes::HTTP_INTERNAL_SERVER_ERROR);
+        }
+
+        $success  = !empty($response['sent']);
```

Comment:
This is really minor, but can you replace the 2 spaces with 1? It exists several times in this file.